parl/remote/remote_manager.py

```diff
@@ -88,6 +101,17 @@ def _wait_for_connection(self):
                 logger.warning('Zmq error, exiting server.')
                 break
 
+    def _check_remote_status(self):
+        while True:
+            for remote_id in list(self.remote_latest_timestamp.keys()):
+                if time.time() - self.remote_latest_timestamp[
+                        remote_id] > 3 * remote_constants.HEARTBEAT_INTERVAL_S:
+                    logger.error(
+                        'Remote object {} is lost, please check if anything wrong in the remote client'
```

Comment:
Remote object {} is lost, please check if anything wrong **happens** in the remote client.

Reply:
done